File: Unidad 2-Estructuras de Control/Ejercicio34.cs
//Equipo 4//
//Fecha: 05/02/2020//
//Un programa que imprima las filas://
//1//
//1 2//
//1 2 3 //
//...//
//1 2 3 ... n//
using System;

namespace Ejercicio34
{
    class Program
    {
        static void Main(string[] args)
        {
            //Entrada//
            int  entrada = 0;//Se define la entrada del programa//
            String salida = "";//se define la salida del programa//

            //Proceso//
            int.TryParse(Console.ReadLine(), out entrada);//Se ingrese la cantidad de filas a imprimir, representadas por la salida//
            Console.WriteLine(" ");
            //Proceso//
            for(int i=0; i<entrada; i++)
            {
                salida = salida + (i+1) + " ";//se realiza la alineacion de los datos//
                //Salida//
                Console.WriteLine(salida);//se imprime la salida de datos alineados por cada fila//

            }
        }
    }
}

File: Unidad 2-Estructuras de Control/Ejercicio34.c
//Equipo 4//
//Fecha: 05/02/2020//
//Un programa que imprima las filas://
//1//
//1 2//
//1 2 3 //
//...//
//1 2 3 ... n//
#include <stdio.h>

int main(void) {
	//Entrada//
	int	N, X, C = 1,i;
	scanf_s("%d", &N);//Se define la entrada//

	//Proceso//
	for (i = 0; i < N; i++) {
		X = 0;
		do {
			X = X++;
			printf("%d", X);
		} while (X<C);
		//salida//
		printf("\n");//se imprime la salida//
		C=C++;
	}
}